GenumKit/Stencil/GenumTemplate.swift

```diff
@@ -41,20 +41,24 @@ open class GenumTemplate: Template {
   }
 }
 
-// Register Genum-specific tags & filters
-open class GenumNamespace: Namespace {
-  public override init() {
-    super.init()
-    self.registerTag("set", parser: SetNode.parse)
-    self.registerFilter("swiftIdentifier", filter: StringFilters.stringToSwiftIdentifier)
-    self.registerFilter("join", filter: ArrayFilters.join)
-    self.registerFilter("lowerFirstWord", filter: StringFilters.lowerFirstWord)
-    self.registerFilter("snakeToCamelCase", filter: StringFilters.snakeToCamelCase)
-    self.registerFilter("snakeToCamelCaseNoPrefix", filter: StringFilters.snakeToCamelCaseNoPrefix)
-    self.registerFilter("titlecase", filter: StringFilters.titlecase)
-    self.registerFilter("hexToInt", filter: NumFilters.hexToInt)
-    self.registerFilter("int255toFloat", filter: NumFilters.int255toFloat)
-    self.registerFilter("percent", filter: NumFilters.percent)
-    self.registerFilter("escapeReservedKeywords", filter: StringFilters.escapeReservedKeywords)
-  }
+// Create Genum-specific namespace including custom tags & filters
+func genumNamespace() -> Namespace {
+  let namespace = Namespace()
+  namespace.registerTag("set", parser: SetNode.parse)
+  namespace.registerTag("ifstrequal", parser: StrEqualsNode.parse)
+  namespace.registerTag("ifnotstrequal", parser: StrEqualsNode.parse_ifnotstrequals)
+  namespace.registerTag("map", parser: MapNode.parse)
+  namespace.registerFilter("swiftIdentifier", filter: StringFilters.stringToSwiftIdentifier)
+  namespace.registerFilter("join", filter: ArrayFilters.join)
+  namespace.registerFilter("lowerFirstWord", filter: StringFilters.lowerFirstWord)
+  namespace.registerFilter("snakeToCamelCase", filter: StringFilters.snakeToCamelCase)
+  namespace.registerFilter("snakeToCamelCaseNoPrefix", filter: StringFilters.snakeToCamelCaseNoPrefix)
+  namespace.registerFilter("titlecase", filter: StringFilters.titlecase)
+  namespace.registerFilter("hexToInt", filter: NumFilters.hexToInt)
+  namespace.registerFilter("int255toFloat", filter: NumFilters.int255toFloat)
+  namespace.registerFilter("percent", filter: NumFilters.percent)
+  namespace.registerFilter("escapeReservedKeywords", filter: StringFilters.escapeReservedKeywords)
+  namespace.registerTag("ifstrequal", parser: StrEqualsNode.parse)
```

Comment:
I don't think this tag or the one below would be necessary with Stencil 0.7, you can use the new `if` statement that takes operators. For example:

```html+django
{% if something == "example" %}

{% endif %}
```

```html+django
{% if something != "example" %}

{% endif %}
```